test-resources/swift/cleanup_rules/input/SampleClass.swift

```diff
@@ -37,9 +36,21 @@ class SampleClass {
     }
     
     func checkOrFalse() {
-        isEnabled = !TestEnum.stale_flag_one.isEnabled || f1()
-        isEnabled = !TestEnum.stale_flag_one.isEnabled || v1
-        isEnabled = f2() || !TestEnum.stale_flag_one.isEnabled 
-        isEnabled = v2 || !TestEnum.stale_flag_one.isEnabled 
+        isEnabled = placeholder_false || f1()
+        isEnabled = placeholder_false || v1
+        isEnabled = f2() || placeholder_false 
+        isEnabled = v2 || placeholder_false 
+    }
+    func checkIfTrueCleanup() {
+        f1()
+        if TestEnum.stale_flag_one.isEnabled {
+            f2()
+        }
+
+        if isEnabled {
+            f2()
```

Comment:
can we have one more longer `if-else` ladder? Can we have isEnabled somewhere in the middle of the ladder?